stdlib/public/Concurrency/Task+immediate.swift.gyb

```diff
@@ -256,7 +257,7 @@ extension Task where Failure == ${FAILURE_TYPE} {
   @discardableResult
   public static func startOnMainActor(
     priority: TaskPriority? = nil,
-    ${OPERATION_PARAM}
+    @_inheritActorContext @_implicitSelfCapture _ operation: __owned @Sendable @escaping @MainActor () async ${THROWS} -> Success
```

Comment:
This is the same as OPERATION_PARAM but moved to the THROWS pattern we use elsewhere in this file